src/components/BookItem/BookItem.js

```diff
@@ -1,24 +1,41 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-// import useDispatch hook
 import { useDispatch } from 'react-redux';
-// import your Action Creators
 import { removeBookAsync } from '../../redux/books/books';
-import styles from './BookItem.module.css';
 
 function BookItem({ book }) {
   const dispatch = useDispatch();
+
   const removeHandler = () => {
     dispatch(removeBookAsync(book.item_id));
   };
 
   return (
-    <div className={styles.bookItem}>
-      <p className="itemCategory">{ book.category }</p>
-      <h2 className="itemTitle">{book.title}</h2>
-      <ul className={styles.itemActions}>
-        <li><button type="button" onClick={removeHandler}>Remove</button></li>
-      </ul>
+    <div className="bg-white mx-20 my-4 p-5 border rounded border-zinc-200 flex flex-wrap">
+      <div className="grow">
+        <p className="text-gray-400 text-sm font-bold">{book.category}</p>
+        <h2 className="text-2xl font-bold">{book.title}</h2>
+        <p className="text-sky-500 text-sm">Frank Herbert</p>
+        <ul className="text-sky-500 text-sm flex flex-row mt-4 gap-x-3">
+          <li><button type="button" onClick="">Comments</button></li>
```

Comment:
[OPTIONAL] I think it's better if we delete the `onClick` events that we are not using yet. This will help us keep the code clean and professional. If at a certain point in the future we decide to implement this functionality we can always add it again.